package.json

```diff
@@ -1,27 +1,27 @@
 {
   "name": "hey-personal-portfolio",
-  "version": "0.1.1",
+  "version": "0.2.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^3.9.1",
-    "@material-ui/icons": "^3.0.2",
-    "@material-ui/styles": "^3.0.0-alpha.9",
-    "@types/cheerio": "^0.22.10",
-    "@types/jest": "^23.3.13",
-    "@types/node": "^10.12.19",
-    "@types/react": "^16.7.22",
-    "@types/react-dom": "^16.0.11",
-    "cheerio": "^1.0.0-rc.2",
+    "@material-ui/core": "4.11.3",
+    "@material-ui/icons": "4.11.2",
+    "@material-ui/styles": "4.11.3",
+    "@types/cheerio": "0.22.23",
+    "@types/jest": "26.0.20",
+    "@types/node": "14.14.20",
+    "@types/react": "17.0.1",
+    "@types/react-dom": "17.0.0",
+    "cheerio": "1.0.0-rc.5",
     "promise-polyfill": "^8.1.0",
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-ga": "^2.5.7",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-ga": "3.3.0",
     "react-plx": "^1.3.11",
-    "react-scripts": "2.1.3",
+    "react-scripts": "3.4.4",
```

Comment:
Unfortunately `yarn start` would stop working with `v4` so upgrading to `v3` only

Reply:
Reference: https://github.com/facebook/create-react-app/issues/9868

Reply:
And no `4.0.1` did not fix it for me (despite there [are](https://github.com/facebook/create-react-app/issues/9868#issuecomment-731966356) [reports](https://github.com/facebook/create-react-app/issues/9868#issuecomment-732016066) that `4.0.1` should fix the problem)